dev-portal/src/components/PageWithSidebar.jsx

```diff
@@ -0,0 +1,122 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react'
+
+// semantic-ui
+import { Button, Icon, Sidebar, Segment, Responsive, Menu } from 'semantic-ui-react'
+
+import _ from 'lodash'
+
+
+export default withCollapsibleMobileSidebar(class PageWithSidebar extends React.Component {
+  render() {
+    const {sidebarContent, content, SidebarPusherProps} = this.props
```

Comment:
Super cool! Can you change PageWithSidebar to accept children instead of content?

Reply:
Absolutely! I was considering that idea myself when coding this.

I'll should be able to make this change late tonight, or Friday afternoon.

Let me know if there's any other changes you'd like to see. :)

Reply:
Updated in 9be4461d